templates/publisher/market.html

```diff
@@ -178,5 +196,18 @@ <h1 class="u-no-margin--top">{{ title }}</h1>
 <script>
```

Comment:
If snap is not uploaded I have an error because the object icon_url is undefined.

One solution could be to add `{% if uploaded %}`  for the script tag?

Reply:
Fixed